styles.css

```diff
@@ -490,10 +488,20 @@ textarea::placeholder {
   border: none;
   border-radius: 0.8rem;
   cursor: pointer;
-  margin: 0 auto;
+  margin-right: 56%;
   margin-top: 5%;
 }
 
+.cont-btn:hover {
+  background-color: rebeccapurple;
+  color: #fff;
+  cursor: pointer;
+}
+
+.cont-btn:active {
+  background-color: #fff;
+}
+

```

Comment:
- [ ] Kindy ensure that you design these pseudo classes according to the given template. `#6070FF` and `#2230D2`
 are the background-colors for hover and active states respectively. 👍 

![DSC_8108](https://github.com/hetrox8/portfolio1/assets/103386183/609aa4f9-5d35-4c16-bffa-f647b0a22efc)

- [ ] Kindly ensure that the button element is aligned properly with other input element according to the design. Implementing this will give your UI a nice interface. 👍